src/rubrix/server/tasks/token_classification/metrics.py

```diff
@@ -333,16 +333,18 @@ def density(value: int, sentence_length: int) -> float:
         return value / sentence_length
 
     @staticmethod
-    def capitalness(value: str) -> str:
+    def capitalness(value: str) -> Optional[str]:
         """Compute capitalness for a string value"""
         value = value.strip()
-        if value.upper() == value:
+        if value.isupper():
             return "UPPER"
-        if value.lower() == value:
+        if value.islower():
             return "LOWER"
         if value[0].isupper():
```

Comment:
In [Explainaboard](http://explainaboard.nlpedia.ai/leaderboard/task-ner/) the capitalness "FIRST" is actually defined as "first_caps represents only the first character of the entity is capital", which is equivalent to `str.istitle()`. I think this is more useful since "typical entities" like PER or LOC are written like this.

Reply:
For this case:

```python 
"Company SL".istitle()
```

The result is `False`, but it should be counted as FIRST, maybe. The docstring for `istitle` says:

> In a title-cased string, upper- and title-case characters may only follow uncased characters and lowercase characters only cased ones.

what's mean that `Peter Parker` should works,  but `The ROCK` does not. I prefer keep the custom check for first capital check.

Reply:
I just checked the source code of explainaboard and we actually do the same ... ok, i will push a commit to this PR in which I update the doc string of this metric to make the definition of each "capitalness" crystal clear.